pkg/proxy/kafka_test.go

```diff
@@ -58,8 +58,17 @@ type proxyTestSuite struct {
 
 var _ = Suite(&proxyTestSuite{})
 
+type identityAllocatorOwnerMock struct{}
+
+func (i *identityAllocatorOwnerMock) UpdateIdentities(added, deleted cache.IdentityCache) {}
+
+func (i *identityAllocatorOwnerMock) GetNodeSuffix() string {
```

Comment:
I wonder if it would help to have a `pkg/testutils/identity` for mocks like this that are used across multiple packages.

(Not required for PR)

Reply:
Upcoming commit will have this fixed!